recipes/argtable2/all/conanfile.py

```diff
@@ -43,15 +50,15 @@ def configure(self):
         del self.settings.compiler.cppstd
 
     def build_requirements(self):
-        if tools.os_info.is_windows and self.settings.compiler != "Visual Studio":
-            self.build_requires("msys2/20200517")
+        if self._settings_build.os == "Windows" and self.settings.compiler != "Visual Studio":
+            self.build_requires("msys2/cci.latest")
 
-        if self.settings.os != "Windows":
+        if self.settings.compiler != "Visual Studio":
             self.build_requires("gnu-config/cci.20201022")
```

Comment:
github doesn't let me suggest a diff, but you can simplify with:

```python
        if self.settings.compiler != "Visual Studio":
            self.build_requires("gnu-config/cci.20201022")
            if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
                self.build_requires("msys2/cci.latest")
```

Reply:
Added your suggestion